s/strimzi-kafka-bridge/redhat_ubi8/build.sh

```diff
@@ -0,0 +1,60 @@
+# ----------------------------------------------------------------------------
+#
+# Package       : strimzi-kafka-bridge
+# Version       : master (default)
+# Source repo   : https://github.com/strimzi/strimzi-kafka-bridge
+# Tested on     : RHEL_8.3
+# Script License: Apache License, Version 2 or later
+# Maintainer    : Krishna Harsha Voora <krishvoor@in.ibm.com>
+#
+# Disclaimer: This script has been tested in root mode on given
+# ==========  platform using the mentioned version of the package.
+#             It may not work as expected with newer versions of the
+#             package and/or distribution. In such case, please
+#             contact "Maintainer" of this script.
+#
+# ----------------------------------------------------------------------------
+
+runtest=0
+
+case "$1" in
+		   # no branch supplied, master is used
+	"")	   BRANCH=""
+		   ;;
+
+	"runtest") BRANCH=""
+		   runtest=1
+		   ;;
+
+	*)	   BRANCH="--branch $1"
+		   if [ "$2" == "runtest" ]
+		   then
+			runtest=1
+     		   fi
+		   ;;
+esac
+
+echo "BRANCH = $BRANCH"
+
+# Clone Repository
+(git clone $BRANCH https://github.com/strimzi/strimzi-kafka-bridge) || (echo "git clone failed" && exit $?)
+cd strimzi-kafka-bridge
+
+# Build
+mvn install -DskipTests
```

Comment:
The intent of build the script was to build the artifacts, so I leveraged `mvn install -DskipTests` which is building and installing the packages. Towards the end of the scripts I am cleaning up the workspace.

FWIW, I can also perform `make docker_build` in case you want me to add this, I can.